modules/yarn.lock

```diff
@@ -14776,6 +14772,15 @@ terser-webpack-plugin@^5.0.3:
     source-map "^0.6.1"
     terser "^5.3.8"
 
+terser@5.3.8:
```

Comment:
Oops. We have two `terser`s...

```
=> Found "terser@5.3.8"
info Reasons this module exists
   - "_project_#@liferay#npm-scripts" depends on it
   - Hoisted from "_project_#@liferay#npm-scripts#terser"
info Disk size without dependencies: "2.95MB"
info Disk size with unique dependencies: "3.94MB"
info Disk size with transitive dependencies: "3.95MB"
info Number of shared dependencies: 4
=> Found "html-minifier-terser#terser@4.7.0"
info This module exists because "_project_#@liferay#npm-scripts#html-webpack-plugin#html-minifier-terser" depends on it.
info Disk size without dependencies: "1.86MB"
info Disk size with unique dependencies: "2.85MB"
info Disk size with transitive dependencies: "2.87MB"
info Number of shared dependencies: 4
=> Found "terser-webpack-plugin#terser@5.4.0"
info This module exists because "_project_#@liferay#npm-scripts#webpack#terser-webpack-plugin" depends on it.
info Disk size without dependencies: "2.96MB"
info Disk size with unique dependencies: "3.95MB"
info Disk size with transitive dependencies: "3.96MB"
info Number of shared dependencies: 4
```

Reply:
I guess the one not from webpack terser plugin is for the minification of JS...

Reply:
Correct.